scripts/deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PACKAGE_VERSION=$(cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g')
```

Comment:
good variable naming here, allows the viewer to not need to understand the shell commands